QuadPaySDK/PaymentWidget/PaymentWidget.swift

```diff
@@ -30,7 +30,47 @@ public final class PaymentWidget: UIView {
                         self.timelapseGraphView.drawTimelapseGraph()
                     }
                     print("Error fetching merchant")
-                }}        }
+                }
+            }
+            
+            WidgetDataService.shared.fetchWidgetData(merchantId: merchantId){
+                (result) in
+                switch result {
+                case .success(let result):
+                    print(result)
+                    
+                    var maxTier: Double = 0
+                    let amountAsFloat  = Double(self.amount) ?? 0.00
```

Comment:
Just a question, is this protected against no amount being sent in?  i.e. assume it will default to 0.00 just want to double check